### PR TITLE
[BFCL Website] Add Search Functionality for Leaderboard and Rmove Multi-Turn Composite Column

### PR DESCRIPTION
**Pull request by Sharonfish**
Added search functionality for leaderboard table and disabled Multiturn Composite Column

### CODE REVIEW

**Comment by CharlieJCJ** [comment]
@Sharonfish Thanks for the PR, can you attach a video recording for this functionality in the PR description? Thanks

File: leaderboard.html [hunk 1]
```diff
@@ -106,13 +106,41 @@ <h2>BFCL Leaderboard</h2>
...
 
-                            2024-10-21 <a
-                                href="https://github.com/ShishirPatil/gorilla/blob/main/berkeley-function-call-leaderboard/CHANGELOG.md">[Change
-                                Log]</a></i></b></span>
+                    <div class="d-flex mt-3 mt-md-0" style="gap: 5px; width: 100%; max-width: 400px;">
+                        <input type="text" id="search-input" class="form-control flex-grow-1" placeholder="Search model names..." />
+                        <button id="search-btn" class="btn btn-primary">Search</button>
+                    </div>
+                </div>
+            </div> -->
```
Comment (CharlieJCJ) [inline leaderboard.html:124]: remove commented code

**Review by HuanzhiMao — APPROVED** [review]
LGTM.  Thanks @Sharonfish!

**Comment by CharlieJCJ** [comment]
LGTM, thanks @Sharonfish

**Review by CharlieJCJ — APPROVED** [review]

**Pull request merged**